src/contributing.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: layouts/base.njk
+---
+
+Welcome! First off; _**THANK YOU**_ for your interest in contributing to Zinc!
+Your time is valuable, so let's get started!
+
+## Ground Rules
+
+Zinc moves forward thanks to two kinds of workers. _Contributors_, who suggest
+and build changes to our software; and _Maintainers_, who decide if, when, and
+how to apply the suggestions. Anyone and everyone can be a Contributor! All we
+ask is that Contributors follow a few [Ground Rules][contributor-ground-rules]
```

Comment:
Linking to the current section seems unnecessary. Or what am I missing?

Reply:
Oops! This was in the introduction previously, but I moved it into the `Ground Rules` heading.